website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Multi-Tenancy Best Practices"
+linkTitle: "Multi-Tenancy"
+weight: 4
+description: This guide describes the best practices concepts to write Operators for Multi-Tenancy solutions.
+---
+
+## Operator Behavioral Guidelines
+
+### Managing Resources
+
+#### Resource Requests
+
+In Multi-tenant clusters, administrators must consider the effects that one tenant's workloads may have on others. 
```

Comment:
this is not just a multitenant problem.  The operator can also compete with its own operands, with other workloads from the same user(if that is considered the same tenant), and with the control plane/core components.

Reply:
I agree 100+
So, since this doc is about multi-tenancy, wdyt about we address it in another doc/topic in the best section practices? suggestion?